make/modules/jdk.compiler/Gendata.gmk

```diff
@@ -99,3 +98,5 @@ $(eval $(call SetupCopyFiles, COPY_TO_LIBS, \
 TARGETS += $(COPY_TO_LIBS)
 
 ################################################################################
+
```

Comment:
This too is provided by the framework, and is not needed in these "snippet" makefiles.

Reply:
These new lines should still be deleted.